frontend/src/metabase/admin/settings/components/EmbeddingSettings/StaticEmbeddingSettings.tsx

```diff
@@ -76,7 +75,7 @@ export function StaticEmbeddingSettings({
         </Box>
         <Box data-testid="embedded-resources">
           <SettingTitle>{t`Manage embeds`}</SettingTitle>
-          <EmbeddedResources key={isStaticEmbeddingEnabled.toString()} />
```

Comment:
We need this. I should have added a comment here. Otherwise, the component won't be reloaded when we toggle the static embedding inside the settings page.

Here's how it should work with `key`

https://www.loom.com/share/057ac461d4384f149815168f5cd7aff7

Reply:
I solved this actually - that's what [Add API endpoints for public + embedded entities](https://github.com/metabase/metabase/pull/48290) was for. Instead of needing to reload things, we're invalidating tags and doing the full RTK query setup for this